src/libutils/oglapphelpers/oglapp.h

```diff
@@ -26,14 +26,14 @@ namespace OCIO_NAMESPACE
 // will appear in the UI so here you specify the name of the window and its size.  For non-GUI
 // apps you may want to use a small size (it does not need to match the size of the image being
 // processed).
-OglApp oglApp("Window Name", windowWidth, windowHeight);
+//
+
+
+// TO DO: to be updated.
```

Comment:
As we now want to merge the pull request when ready, you must update the comment.